src/scroll-to-top.css

```
@@ -0,0 +1,93 @@
+@keyframes scrollToTopFadeOut {
+  from {
+    opacity: 1;
+    transform: translateY(0) scale(1);
+  }
+  to {
+    opacity: 0;
+    transform: translateY(20px) scale(0.9);
+  }
+}
+
+@import url("scroll-to-top.css");
```

Stylelint (17.14.0)

[error] Expected "url("scroll-to-top.css")" to be ""scroll-to-top.css"" (import-notation)

(import-notation)